src/routes/board.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const Board = require("../models/board.model");
+const router = express.Router();
+var {validationResult} = require('express-validator');
+const validator = require("../middlewares/validator");
+//get all board
+router.get('/', async function(req, res, next) {
+    try {
+        const boards = await Board.find()
+        .populate({ path : "created_by" });
+        res.status(200).json({
+            data: boards,
+            message : "get all board successfully !",
+        }) 
+    } catch (error) {
+        res.status(500).json({
```

Comment:
- Status Code must be 400 (Bad Request)
Reference: [(https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)](url)